lib/tasks/auto_whitelist.rb

```diff
@@ -0,0 +1,57 @@
+desc <<-DESC
+  Whitelists all nodes edges based on a sparse graph
+DESC
+
+def add_method(el, method)
+  unless el.graph_methods.include?(method)
+    el.graph_methods << method
+  end
+end
+
+task auto_whitelist: :environment do
```

Comment:
Does this not belong in ETSource?

Reply:
Yes, it should. I'll add it to the list, stating that ETsource probably needs some TLC first when it comes to it's rake tasks.